ERCS/erc-7551.md

```diff
@@ -0,0 +1,267 @@
+```
+
+### Interface
+```solidity
+interface IERCXXXX {
+	
+	// Events
+	event TokensIssued(address to, uint256 amount, bytes data);
+	event TokensDestroyed(address tokenHolder, uint256 amount, bytes data);
+	event ForcedTransfer(address indexed from, address indexed to, uint256 amount, bytes data);
+	event TokensFrozen(address indexed tokenHolder, uint256 amount, bytes data);
+	event TokensUnfrozen(address indexed tokenHolder, uint256 amount, bytes data);
+	event SetPaperContractHash(uint256 paperContractHash);
+	event SetMetaDataJSON(string metaDataJSON);
+	
+	// View functions
+	function activeBalanceOf(address tokenHolder) external view returns (uint256);
+	function frozenBalanceOf(address tokenHolder) external view returns (uint256);
+	function paused() external view returns (bool);
+	function paperContractHash() external view returns (uint256);
+	function metaDataJSON() external view returns (string memory);
+	function canTransfer(address to, uint256 amount) external view returns (bool);
+	function canTransferFrom(address from, address to, uint256 amount) external view returns (bool);
+	
+	// Operator functions
+	function issue(address to, uint256 amount, bytes calldata data) external;
+	function destroyTokens(address tokenHolder, uint256 amount,  bytes calldata data) external;
+	function forceTransferFrom(address from, address to, uint256 amount, bytes calldata data) external;
+	function freezeTokens(address tokenHolder, uint256 amount, bytes calldata data) external;
+	function unfreezeTokens(address tokenHolder, uint256 amount, bytes calldata data) external;
+	function pauseTransfers(bool _paused) external;
+	function setPaperContractHash(uint256 _paperContractHash) external;
+	function setMetaDataJSON(string calldata _metaDataJSON) external;
+}
+```
+
+
+## Rationale
+
+This standard is the result of the standardization working group of the German Federal Association of Crypto Registrars. It’s based on the smart contract implementations of its members. It contains terminology and abstract ideas also published in the [EIP-1411](./eip-1411.md) and [EIP-3643](./eip-3643.md) standard proposals. Both proposals were considered as alternatives to drafting this standard but were rejected because they seemed not complete or contained features for specific use cases not common to all association members. 
+
+This standard should not be understood to be a guideline for developing security token implementations. For a full security token implementation, we expect this standard to be combined with the underlying token itself (e.g. based on [EIP-20](./eip-20.md) or [EIP-1155](./eip-1155.md))), permission management and authorization logic for operators, different mechanisms to determine the compliance of a specific token transfer, as well as mechanisms to upgrade the token smart contract logic. In contrast, this interface standard describes minimum requirements for the token smart contract representing a crypto security.
```

Comment:
There's a mix of non-Rationale appropriate content with appropriate content in this section. The Rationale section should explain technical choices made within the proposal itself, not who made the proposal or usage instructions.